apps/examples/testcase/le_tc/kernel/tc_libc_sched.c

```diff
@@ -90,6 +123,8 @@ int libc_sched_main(void)
 {
 	tc_libc_sched_sched_get_priority_max();
 	tc_libc_sched_sched_get_priority_min();
-
+#ifndef CONFIG_CANCELLATION_POINTS
```

Comment:
I think we should add TC for CONFIG_CANCELLATION_POINTS=y case.
But when CONFIG_CANCELLATION_POINTS is enabled, kernel/task/task_setcanceltype.c will be compiled, so you can add this TC to sched : tc_sched_task_setcanceltype()